gcc.symbols.imp

```diff
@@ -121,6 +121,10 @@
   { "symbol": [ "MAXHOSTNAMELEN", "private", "<protocols/timed.h>", "public"] },
   { "symbol": [ "SIGABRT", "private", "<signal.h>", "public"] },
   { "symbol": [ "SIGCHLD", "private", "<signal.h>", "public"] },
+  { "symbol": [ "va_start", "private", "<stdarg.h>", "public"] },
```

Comment:
Could you alpha-sort these (including `va_list`)? And same in .cc file.

Reply:
Ahh, sorry!  I try to always use alpha sort, but I guess I blindly followed the manual page order.  Will do.

Reply:
Done.